### DISCUSSION TITLE
Video performance in 2022.44

### DISCUSSION THREAD

**Discussion by themajesticking** [tesla-android/issue-tracker#126] [Q&A] [answered]
Hey guys. After eagerly awaiting 2022.44 I happily disconnected the Geekworm capture card and threw it in a drawer and installed 2022.44 on my 8GB Pi4. It's working great and it's nice having higher video resolution in carplay but I've been finding the video performance to be a bit choppy. 

In 2022.38.1 I was able to play Youtube videos in the web browser or Youtube app and the video was nice and fluid but with the latest version which doesn't require the Geekworm capture card (2022.44) I'm getting something like 8-10fps from Youtube videos. I know I can overclock the Pi but with 2022.38.1 I found the video performance was somehow better without the overclock. I suppose I could give it a try with this new version but I wanted to get some feedback from other 2022.44 users. Has anyone else run into any issues?

Forgot to mention that I'm using the Ice Tower so I don't think it's a heat-related issue.

**Comment by adamChrystie** [comment]
By coincidence I viewed YouTube via the pi www browser last night. I was just curious. Playback was poor around 8-15fps, very choppy even at lower than 720P.  Videos max res was always 720P too.

My viewing environment:
I am using 2022.44 , no geekworm hardware.
No carlinkit dongle plugged in. 
no lte modem plugged in. 
Pi connected to my home internet via Ethernet connection. 250mbps down/50 up. Pi output was viewed on my 27” iMac 5K display via Wi-Fi. 
youtube video & sound output worked. 
My pi is over clocked using the suggested settings.
I have an ice cooler tower.
Pi has 8gb ram.

**Comment by mikegapinski** [comment, marked as answer]
2022.44.xx uses higher resolution display, since video is decoded on the CPU now a low streaming performance is expected until I release the hardware decoding component for h264 - I have it working in a proof of concept state, there are still some artefacts on decoding. With this new feature video playback will consume like 5% of cpu instead of 60/70. 

2022.44.xx is not heating up as much thanks to the changes in video layer, the capture card software was much more heavy on the system

**Reply by sw1002** [reply]
Having the same issues here, though video playback on YT installed as apk works but Disney+ and Prime Video just playback audio and a dark screen occurs in the tesla browser while playing back the video. When I connect HDMI directly to the Pinthe playback video appears correctly. An issue with the acreen mirroring through 9.9.0.1?
